2023-11-30 10:06 WARNING Not Found: / 
2023-11-30 10:06 WARNING Not Found: /favicon.ico 
2023-11-30 10:39 WARNING Not Found: /admin/app 
2023-11-30 10:40 INFO /Users/yhlin/yhWorkspace/Web/goodvilla_web/app/admin.py changed, reloading. 
2023-11-30 10:49 WARNING Not Found: / 
2023-11-30 10:49 WARNING Not Found: /api 
2023-11-30 10:49 WARNING Not Found: /accounts/login/ 
2023-11-30 10:50 WARNING Not Found: /accounts/login/ 
2023-11-30 10:50 INFO /Users/yhlin/yhWorkspace/Web/goodvilla_web/app/urls.py changed, reloading. 
2023-11-30 10:50 WARNING Not Found: /app/ 
2023-11-30 10:50 WARNING Not Found: /app/ 
2023-11-30 10:56 INFO /Users/yhlin/yhWorkspace/Web/goodvilla_web/app/urls.py changed, reloading. 
2023-11-30 11:01 WARNING Not Found: / 
2023-11-30 11:02 ERROR Internal Server Error: /app/ 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/core/handlers/base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/contrib/auth/decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/app/views.py", line 35, in index
    return  render(request,'app/index.html',{'major':major})
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/template/loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: app/index.html
2023-11-30 11:11 WARNING Not Found: /accounts/login/ 
2023-11-30 11:12 WARNING Not Found: /app/sub_view/1 
2023-11-30 14:11 INFO [] 
2023-11-30 14:12 INFO [] 
2023-11-30 14:30 INFO [] 
2023-11-30 14:49 INFO [] 
2023-11-30 14:49 INFO [] 
2023-11-30 14:53 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 50, 'daily_bonus': 0, 'daily_moods': -2}] 
2023-11-30 14:53 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 100, 'daily_bonus': 0, 'daily_moods': -4}] 
2023-11-30 14:53 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 100, 'daily_bonus': 0, 'daily_moods': -6}] 
2023-11-30 14:54 ERROR Internal Server Error: /app/api/workrecord/1 
Traceback (most recent call last):
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/core/handlers/exception.py", line 55, in inner
    response = get_response(request)
               ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/utils/deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/middleware/common.py", line 108, in process_response
    return self.response_redirect_class(self.get_full_path_with_slash(request))
                                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/yhlin/yhWorkspace/Web/goodvilla_web/venv/lib/python3.11/site-packages/django/middleware/common.py", line 87, in get_full_path_with_slash
    raise RuntimeError(
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 127.0.0.1:8000/app/api/workrecord/1/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
2023-11-30 14:54 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 200, 'daily_bonus': 0, 'daily_moods': -8}] 
2023-11-30 14:55 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 200, 'daily_bonus': 0, 'daily_moods': -8}] 
2023-11-30 15:52 WARNING Not Found: / 
2023-11-30 15:59 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 250, 'daily_bonus': 0, 'daily_moods': -10}, {'major_id': 3, 'item': '住', 'daily_spend': 50, 'daily_bonus': 0, 'daily_moods': -2}] 
2023-11-30 15:59 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 250, 'daily_bonus': 0, 'daily_moods': -10}, {'major_id': 3, 'item': '住', 'daily_spend': 170, 'daily_bonus': 0, 'daily_moods': -1}] 
2023-11-30 16:00 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 250, 'daily_bonus': 0, 'daily_moods': -10}, {'major_id': 3, 'item': '住', 'daily_spend': 170, 'daily_bonus': 0, 'daily_moods': -1}] 
2023-11-30 16:00 INFO [{'major_id': 1, 'item': '食', 'daily_spend': 250, 'daily_bonus': 0, 'daily_moods': -10}, {'major_id': 3, 'item': '住', 'daily_spend': 170, 'daily_bonus': 0, 'daily_moods': -1}, {'major_id': 5, 'item': '育', 'daily_spend': 240, 'daily_bonus': 2000, 'daily_moods': 4}] 
